script/brave_license_helper.py

```diff
@@ -146,6 +149,21 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
             "URL": "https://github.com/ronaldoussoren/macholib",
             "License": "MIT",
         },
+        os.path.join('brave', 'third_party', 'ios_deps', 'JitsiMeet'): {
```

Comment:
These look like they are "vendored in" as opposed to pulled them from external repos in the build step, no?

If so, then we don't need to add them here and instead we can add ` README.chromium` and `LICENSE` files in `third_party/ios_deps/{GRDWireGuardKit,JitsiMeet,Static}`. In fact `Static/LICENSE` already exists.

Reply:
Here's an example from the Android side: https://github.com/brave/brave-core/tree/master/third_party/android_deps/libs/com_github_bumptech_glide

Reply:
Fixed in https://github.com/brave/brave-core/pull/21553/commits/d05f0c407975d4c34d611641a675edf026730a4d